lab-maria/test/test-modules.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+// I've left a bunch of commented code intentionally in the hopes that someone
+// will eventually help me figure out why I wasn't getting the result I wanted
+// if it's really annoying and docks points I'll ta
+
+const mod = require('../lib/modules.js');
+const chai = require('chai');
+const expect = chai.expect;
+const chaiParam = require('chai-param');
+//const param = chaiParam.param;
+chai.use(chaiParam);
+const sinonChai = require('sinon-chai');
+const sinon = require('sinon');
+chai.use(sinonChai);
+
+describe('Person', function(){
+  it('should be an object constructor', function(){
+    expect(mod.Person.prototype).to.be.a('object');
+  });
+});
+
+describe('call', function(){
+  it('should call the function with the given list of arguments', function(){
+    let scott = new mod.Person('scott');
+    var spy = sinon.spy(mod.schedule);
```

Comment:
sinon will be useful when we get into front-end as well. diggin the fact that you're searching for what's useful and researching!